release.sh

```diff
@@ -114,19 +117,20 @@ rm -f package.json.bak
 
 # Update the version number in CDN URLs included in the README files,
 # and regenerate the Subresource Integrity hash for these files.
-node update-sri.js "${VERSION}" README.md contrib/*/README.md
+node update-sri.js "${VERSION}" README.md contrib/*/README.md dist/README.md
 
 # Make the commit and tag, and push them.
-git add package.json bower.json
+git add package.json bower.json README.md contrib/*/README.md dist/README.md
 git commit -n -m "v$VERSION"
+git diff --stat --exit-status # check for uncommitted changes
```

Comment:
Does the script exit prematurely if `git diff --stat --exit-status` is non-zero?